### ISSUE TITLE
can't play m3u stream

### ISSUE THREAD

**Issue by veso266** [Koenvh1/ets2-local-radio#54]
not sure what can be done but when adding canadian stations I came across quite a few stations that uses DASH Streams (here is Spice Radio from Vancouver for instance: http://d2e1wsg2gpoq0n.cloudfront.net/livecf/CJRJ/playlist.m3u8 ) using Firefox and it aphears HLS is supported and VLC plays it fine too

**Comment by Koenvh1** [comment]
Should work, if it doesn't, prepend it with https://cors-anywhere.herokuapp.com/ (look at the other stations for which it is done as well)

It should work then.

**Comment by veso266** [comment]
nope it didn't help (for some reason they show green in stationtest.html) (probably firefox 49 problem

**Comment by Koenvh1** [comment]
If they show up green in stationtest, then they work. It might be that the stream takes a few seconds to load, or that your volume is too low.

**Comment by veso266** [comment]
nope they don't work in chrome either (chrome 62)
do they work for you?

**Comment by Koenvh1** [comment]
Yup, I've located the issue. Sadly, I have no solution for it at the moment.

**Comment by veso266** [comment]
OK will wait and add more stations in Canada then
PS: What is the issue?

**Comment by Koenvh1** [comment]
Well, just Cloudfront streams seem to be malfunctioning. The cause is that it requires signed cookies while returning a `Access-Control-Allow-Origin: *` header.
